lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -39,6 +39,19 @@
       <p>${_("Click to generate a CSV file of all proctored exam results in this course.")}</p>
       <p><input type="button" name="proctored-exam-results-report" value="${_("Generate Proctored Exam Results Report")}" data-endpoint="${ section_data['list_proctored_results_url'] }"/></p>
     %endif
+
+    <p>${_("Enter a problem URL, then click to generate a CSV file that lists all student answers to a given problem.")}</p>
```

Comment:
That sounds good to me!

On Tue, Aug 18, 2015 at 9:25 AM, Alison Hodges <notifications@github.com>
wrote:

> In lms/templates/instructor/instructor_dashboard_2/data_download.html
> <https://github.com/edx/edx-platform/pull/9147#discussion_r37296725>:
>
> > @@ -39,6 +39,19 @@
> >        <p>${_("Click to generate a CSV file of all proctored exam results in this course.")}</p>
> >        <p><input type="button" name="proctored-exam-results-report" value="${_("Generate Proctored Exam Results Report")}" data-endpoint="${ section_data['list_proctored_results_url'] }"/></p>
> >      %endif
> > +
> > +    <p>${_("Enter a problem URL, then click to generate a CSV file that lists all student answers to a given problem.")}</p>
>
> I'm a big fan of putting what it does first, followed by how to do it (so
> you'll figure out more quickly if that's what you're trying to find). How's
> about,
> "To generate a CSV file that lists all student answers to a given problem,
> enter the location of the problem (from its Staff Debug Info)."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/9147/files#r37296725>.
>